### PR TITLE
Tysen radovich assignment2

### PR DESCRIPTION
**Pull request by tyrado**

### CODE REVIEW

File: problems/kattis/helloWorld.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print("hello World!")
+j=input("type what you want: ")
+print(j)
```
Comment (wyatt-benson) [inline problems/kattis/helloWorld.py]: Looks Good

File: problems/kattis/R2/Python/tyrado/r2.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    
+    if i <= 1000 and i>= -1000 and j>=-1000 and j<=1000:
+        k =  0 
+        k=((j*2)-i)
+        print(k);
+        j= (i+k)/2
+     
+    else:
+        pass    
+    return(k)
```
Comment (wyatt-benson) [inline problems/kattis/R2/Python/tyrado/r2.py]: Why is k being returned if it is printed at the end of function r2?

File: problems/kattis/R2/Python/tyrado/r2.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    print(k)
+
```
Comment (wyatt-benson) [inline problems/kattis/R2/Python/tyrado/r2.py:17]: Why does the function return and print k after the if statements? The answer prints out in the if statement and the second print is negated by an unnecessary return

File: problems/kattis/R2/Python/tyrado/r2.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+def r2():
+    i=(int( input("put in your numbers: ")))
+    j=(int(input("put in number 2: ")))
+    print(i ,j)
+    
+    if i <= 1000 and i>= -1000 and j>=-1000 and j<=1000:
+        k =  0 
+        k=((j*2)-i)
+        print(k);
+        j= (i+k)/2
```
Comment (wyatt-benson) [inline problems/kattis/R2/Python/tyrado/r2.py:11]: j= equation seems unnecessary. Through substitution with the k = equation it simplifies to j = j.

**Review by wyatt-benson — APPROVED** [review]
+10
runs but has some redundant statements

**Review by wyatt-benson — APPROVED** [review]
+10
runs but has some redundant statements
Screenshot 2020-02-20 at 3 07 05 PM

**Comment by sleddog** [comment]
Contains some extra files not required by assignment.  Please clean up:
`problems/kattis/helloWorld.py`

File: problems/kattis/R2/Python/tyrado/r2.py (hunk 4 above)
Comment (tyrado) [reply problems/kattis/R2/Python/tyrado/r2.py:11]: I had it in their to make sure I had the k equation right

**Review by sleddog — COMMENTED** [review]
There still seems to be some extra files in here referring to Hello World.  Did you mean to have these and the R2 solution?

**Pull request closed without merging**